backend/src/nodes/onnx_nodes.py

```diff
@@ -47,18 +48,32 @@ def run(self, path: str) -> Tuple[ort.InferenceSession, str]:
 
         model_as_string = model.SerializeToString()  # type: ignore
 
-        session = ort.InferenceSession(
-            model_as_string,
-            providers=[
-                "CPUExecutionProvider"
-                if os.environ["device"] == "cpu"
-                else "CUDAExecutionProvider"
-            ],
-        )
-
         basename = os.path.splitext(os.path.basename(path))[0]
 
-        return session, basename
+        return model_as_string, basename
```

Comment:
The type of `run` still says `ort.InferenceSession`. Is that correct?

Reply:
Good catch.